tests/zfs-tests/include/libtest.shlib

```diff
@@ -3269,7 +3269,7 @@ function zed_start
 {
 	if is_linux; then
 		# ZEDLET_DIR=/var/tmp/zed
-		if [[ ! -d /var/tmp/zed ]]; then
+		if [[ ! -d $ZEDLET_DIR ]]; then
 			log_must $MKDIR  /var/tmp/zed
```

Comment:
You'll want to fix the other hard coded instances of `/var/tmp/zed` used in the proposed patch.